docs/changes/v11.4.0/Server-Changes.adoc

```diff
@@ -0,0 +1,7 @@
+= Server Changes =
+
+== <subsystem>-add-user now accepts attributes as an option ==
+
+The `<subsystem>-add-user` methods accept a new option `--attributes` which takes a comma separated string of single-valued attributes as an argument, for example:
+
+`"ham:spam,foo:bar"`
```

Comment:
I'd suggest showing the entire `pki-server <subsystem>-add-user` command instead of just the value.

Reply:
Ack.